src/App.tsx

```diff
@@ -12,161 +18,23 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+          <NavLink className={getLinkClass} to="/">
```

Comment:
```suggestion
          <NavLink 
            className={getLinkClass} 
            to="/"
          >
```
be consistent here with props
Not a mistake thought.